https://issues.redhat.com/browse/ACM-14070 (#7039)
open-cluster-management/rhacm-docs: release_notes/acm_whats_new.adoc

[#dr4hub-whats-new]
== Backup and restore

> * You now have a scenario where you can run a disaster recovery test. By simulating a disaster, you can practice the following actions: restoring hub data on a new hub cluster, verifying that data has been recovered, and returning to the initial hub cluster by using the primary hub cluster as a the active hub cluster. See link:../business_continuity/backup_restore/backup_return_hub.adoc#return-initial-hub[Returning to the initial hub cluster after a restore].

Reviewer: ```suggestion
* You now have a scenario where you can run a disaster recovery test. By simulating a disaster, you can practice the following actions: restoring hub cluster data on a new hub cluster, verifying that data is recovered, and returning to the initial hub cluster by using the primary hub cluster as a the active hub cluster. See link:../business_continuity/backup_restore/backup_return_hub.adoc#return-initial-hub[Returning to the initial hub cluster after a restore].
```
